app/backend/src/clj/teet/file/file_commands.clj

```diff
@@ -85,6 +86,28 @@
     {:url (file-storage/upload-url key)
      :file (d/pull (:db-after res) '[*] file-id)}))
 
+(defcommand :file/upload-user-attachment
+  {:doc "Upload attachment file and optionally attach it to entity."
```

Comment:
Here we could document that we
- attach the file to user AND
- add it to the key person "application"

Reply:
This could even be named something like `upload-key-person-file` since this can't be used to just upload a file to the user.

Reply:
Renamed command and doc